stream_alert/shared/description.py

```diff
@@ -67,16 +82,25 @@ def parse(cls, rule_description):
             dict: A dict mapping fields to lists of strings, each corresponding to a line belonging
                   to that field. All field names are lowercase.
         """
-        tokens = rule_description.split('\n')
+        rule_description = '' if not rule_description else rule_description
+        tokens = rule_description.strip().split('\n')
```

Comment:
We may `strip` after `split`, because
`The strip() method returns a copy of the string with both leading and trailing characters removed (based on the string argument passed).` So after split, the substring may contains leading or trailing characters.

Reply:
Doesn't `split` return an array?

Reply:
I think she means on the values within the list returned by `split`.. like:

`tokens = [line.strip() for line in rule_description.strip().split('\n')]`

Reply:
Python is :magic: